12.LOCO_BULK_MLP/0.data_splits/scripts/0.LOCO_data_split.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import argparse
+import pathlib
+
+import pandas as pd
+import toml
+
+# In[2]:
+
+
+argparser = argparse.ArgumentParser()
+argparser.add_argument("--cell_type", default="all")
+
+args = argparser.parse_args()
+
+cell_type = args.cell_type
+
+
+# In[3]:
+
+
+# set path to the import data
+data_path = pathlib.Path(
+    f"../../../data/{cell_type}_preprocessed_sc_norm_aggregated.parquet"
+)
+# results path
+results_path = pathlib.Path("../results/").resolve()
+results_path.mkdir(exist_ok=True)
+
+# dataframe with only the labeled data we want (exclude certain phenotypic classes)
+data_df = pd.read_parquet(data_path)
+data_df.head()
```

Comment:
I am unsure if this code comment is correct; from my understanding, this data doesn't include phenotypic classes (probably an artifact of the phenotypic profiling repo).

Recommend removing this/updating to reflect this dataset accurately.